topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -82,7 +82,7 @@ has been sequenced using 454 GS FLX Titanium. For the tutorial, the original fas
 
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Import from [Zenodo](https://zenodo.org/record/815875) or from the data library (in "Analyses of metagenomics data") the files
+> 1. Import from [Zenodo](https://zenodo.org/record/815875) or from the data library (in GTN-Material -> "Analyses of metagenomics data") the files
```

Comment:
`GTN-Material -> "Analyses of metagenomics data"`

Maybe it's just a detail but the path seems like that:
`GTN - Material -> Metagenomics -> Analyses of metagenomics data - The global picture`

Reply:
@erxleben ...?